activities.json

```
@@ -1476,11 +1476,11 @@
     "id": "webmidi",
     "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/API/MIDIAccess",
     "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=836897",
-    "mozPosition": "under consideration",
-    "mozPositionDetail": null,
+    "mozPosition": "positive",
+    "mozPositionDetail": "This specification is a reasonable technical design for exposing MIDI devices to JavaScript, which has some demonstrated use-cases in the market. MIDI devices are not universally hardened against adversarial input (especially when sysex is involved) so we don't believe this API is safe to expose in the casual and low-trust context of ordinary websites. We therefore only expose it in Firefox if the user has installed a site-specific add-on to explicitly extend the base web platform with high-trust capabilities ordinarily reserved for installed software.”,
```

Smart quote was added instead of regular one
```suggestion
    "mozPositionDetail": "This specification is a reasonable technical design for exposing MIDI devices to JavaScript, which has some demonstrated use-cases in the market. MIDI devices are not universally hardened against adversarial input (especially when sysex is involved) so we don't believe this API is safe to expose in the casual and low-trust context of ordinary websites. We therefore only expose it in Firefox if the user has installed a site-specific add-on to explicitly extend the base web platform with high-trust capabilities ordinarily reserved for installed software.",
```